src/desktop/apps/experimental-app-shell/server.tsx

```diff
@@ -68,7 +60,10 @@ app.get(
         bodyHTML,
         headTags,
       } = await buildServerApp({
-        context: buildServerAppContext(req, res),
+        context: buildServerAppContext(req, res, {
+          EXPERIMENTAL_APP_SHELL:
+            res.locals.sd.CLIENT_NAVIGATION_V3 === "experiment",
```

Comment:
#minor I think this can be hard-coded to `true`, since this is already the experimental app server-side handler.

Reply:
I wanted to be explict in the sense that `EXPERIMENTAL_APP_SHELL` is a direct result of the running A/B test `CLIENT_NAVIGATION_V3`

Reply:
Will update tho